Sources/ArcGISToolkit/Components/Popups/PopupView.swift

```diff
@@ -89,11 +88,19 @@ public struct PopupView: View {
                     Button(action: {
                         isPresented?.wrappedValue = false
                     }, label: {
-                        Image(systemName: "xmark.circle")
+                        let imageName: String
+#if !os(visionOS)
+                        imageName = "xmark.circle"
+#else
+                        imageName = "xmark"
+#endif
+                        return Image(systemName: imageName)
```

Comment:
```suggestion
                        Image(systemName: "xmark")
#if !os(visionOS)
                            .symbolVariant(.circle)
#endif
```

Reply:
This modifier could actually be moved down a few more lines into the next conditional compilation block. It doesn't need to go on the image view directly.

Reply:
It looks like all the view modifiers on the image can be moved down to the next conditional compilation block. None of them are needed on visionOS and they are still working as expected on iOS.